### PR TITLE
Add megagta

### PR DESCRIPTION
**Pull request by mdehollander**
- [x] I have read the guidelines above.
- [x] This PR adds a new recipe.
- [ ] This PR updates an existing recipe.
- [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by mdehollander** [comment]
Any advice on how to get this build completed? On my local system is works with conda build and has this output:

```
TEST START: megagta-0.1_alpha-0
updating index in: /home/mattias/miniconda3/conda-bld/linux-64
updating index in: /home/mattias/miniconda3/conda-bld/noarch

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    megagta-0.1_alpha          |                0         669 KB  local

The following NEW packages will be INSTALLED:

    hmmer:      3.1b2-1       bioconda
    megagta:    0.1_alpha-0   local   
    openssl:    1.0.2j-0              
    pip:        8.1.2-py27_0          
    python:     2.7.12-1              
    readline:   6.2-2                 
    setuptools: 27.2.0-py27_0         
    sqlite:     3.13.0-0              
    tk:         8.5.18-0              
    wheel:      0.29.0-py27_0         
    zlib:       1.2.8-3               

fatal: No names found, cannot describe anything.
+ source /home/mattias/miniconda3/bin/activate /home/mattias/miniconda3/conda-bld/megagta_1475066512259/_t_env
+ /bin/bash -x -e /home/mattias/miniconda3/conda-bld/megagta_1475066512259/test_tmp/run_test.sh
+ megagta.py -v
TEST END: megagta-0.1_alpha-0
# If you want to upload this package to anaconda.org later, type:
#
# $ anaconda upload /home/mattias/miniconda3/conda-bld/linux-64/megagta-0.1_alpha-0.tar.bz2
#
# To have conda build upload to anaconda.org automatically, use
# $ conda config --set anaconda_upload yes
```
However, the build at travis fails

```
TEST START: megagta-0.1_alpha-0
updating index in: /opt/conda/pkgs
updating: cloog-0.18.0-0.tar.bz2
updating: ant-1.9.6-1.tar.bz2
updating: gmp-5.1.2-2.tar.bz2
updating: isl-0.12.2-0.tar.bz2
updating: autoconf-2.69-0.tar.bz2
updating: gcc-4.8.5-6.tar.bz2
updating: mpfr-3.1.2-0.tar.bz2
updating: mpc-1.0.1-0.tar.bz2
updating: m4-1.4.17-0.tar.bz2
updating: automake-1.14-0.tar.bz2
updating: java-jdk-8.0.92-1.tar.bz2
updating index in: /opt/conda/conda-bld/linux-64
updating index in: /opt/conda/conda-bld/noarch

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    hmmer-3.1b2                |                1         5.5 MB  bioconda
    openssl-1.0.2j             |                0         3.2 MB  anaconda
    readline-6.2               |                2         606 KB  anaconda
    sqlite-3.13.0              |                0         4.0 MB  anaconda
    tk-8.5.18                  |                0         1.9 MB  anaconda
    zlib-1.2.8                 |                3         101 KB  anaconda
    python-2.7.12              |                1        12.1 MB  anaconda
    megagta-0.1_alpha          |                0           5 KB  local
    setuptools-27.2.0          |           py27_0         521 KB  anaconda
    wheel-0.29.0               |           py27_0          81 KB  anaconda
    pip-8.1.2                  |           py27_0         1.5 MB  anaconda
    ------------------------------------------------------------
                                           Total:        29.4 MB

The following NEW packages will be INSTALLED:

    hmmer:      3.1b2-1       bioconda
    megagta:    0.1_alpha-0   local   
    openssl:    1.0.2j-0      anaconda
    pip:        8.1.2-py27_0  anaconda
    python:     2.7.12-1      anaconda
    readline:   6.2-2         anaconda
    setuptools: 27.2.0-py27_0 anaconda
    sqlite:     3.13.0-0      anaconda
    tk:         8.5.18-0      anaconda
    wheel:      0.29.0-py27_0 anaconda
    zlib:       1.2.8-3       anaconda

TESTS FAILED: megagta-0.1_alpha-0

/home/travis/build.sh: line 112:  3177 Terminated              travis_jigger $! $timeout $cmd

travis_time:end:146ffe28:start=1475068332634099461,finish=1475068705231790375,duration=372597690914
[0K
[31;1mThe command "travis_wait 75 scripts/travis-run.sh" exited with 1.[0m

Done. Your build exited with 1.
```

File: recipes/megagta/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package:
+  name: megagta
+  version: "0.1_alpha"
+source:
+  git_url: https://github.com/HKU-BAL/megagta.git
```
Comment (bgruening) [inline recipes/megagta/meta.yaml]: please use a tarball if possible

File: recipes/megagta/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  git_rev: v0.1-alpha
+requirements:
+  build:
+    - automake
+    - autoconf
+    - m4
+    - gcc
```
Comment (bgruening) [inline recipes/megagta/meta.yaml]: please use gcc  # [linux] and llvm # [osx] as build deps and libgcc # [linux] as run time dep

File: recipes/megagta/build.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#strictly use anaconda build environment
+CC=${PREFIX}/bin/gcc
```
Comment (bgruening) [inline recipes/megagta/build.sh]: This should not be needed. Everything in PREFIX/bin should be available.

**Pull request merged**